### DISCUSSION TITLE
Is there a way to build a static site with single locale

### DISCUSSION THREAD

**Discussion by abhisheksatre** [vinissimus/next-translate#1046] [Q&A]
I am trying to build a static Next.js site in the `mr-IN` locale. However, as per the Next.js documentation on [Internationalization](https://nextjs.org/docs/pages/building-your-application/routing/internationalization#how-does-this-work-with-static-generation), it seems that i18n is not supported in static builds.

I was wondering if there is a way to build a static site with a single locale (mr-IN). Is there any workaround or configuration change that can help me achieve this?

Any help or guidance in this regard would be greatly appreciated. 

**What version of this package are you using?**
2.0.2

**What operating system, Node.js, and npm version?**
Mac OS, Node: v18.15.0, NPM: v9.5.0

Thank you.

**Comment by aralroca** [comment]
Using the i18n routing of Next.js is not possible right now.
 
Probably you can do it by [disabling](https://github.com/aralroca/next-translate/blob/3daeb01253fab1770260767a934c9b875290f29c/examples/without-loader/i18n.js#LL4C8-L4C16) the webpack plugin and instead of using i18n routing, implement a dynamic route `[lang]/your-page.ts`, and then using the [I18nProvider ](https://github.com/aralroca/next-translate#i18nprovider) inside.

**Comment by abhisheksatre** [comment]
Thank you @aralroca for the prompt response. 

Would it be feasible to enforce the use of a specific locale during static build generation? My preference is to use the same locale for all routes rather than doing translations on a per-route basis.
